tooling/api/src/fs.ts

```diff
@@ -142,7 +142,7 @@ async function readTextFile(
   return invokeTauriCommand<number[]>({
```

Comment:
this can/should be string and with that the `TextDecoder` stuff in L149 can be removed too.

Reply:
nice catch, totally missed that one.